### DISCUSSION TITLE
Don't know how to turn this value into a node

### DISCUSSION THREAD

**Discussion by tssobe** [storybookjs/storybook#13987] [Help]
When running build-storybook I get:

 ./modules/stories/asButtonExample.stories.js
WARN Module build failed (from ./node_modules/babel-loader/lib/index.js):
WARN Error: C:\works\webapp-as\modules\stories\asButtonExample.stories.js: don't know how to turn this value into a node
WARN     at valueToNode (C:\works\webapp-as\node_modules\@babel\types\lib\converters\valueToNode.js:90:9)

I have tried all options I have found on google, but nothing solves the issue.

As a result , I am not able to see my new stories.

Any idea on how to solve it?

**Comment by shilman** [comment]
Do you have a repro repo you can share?
